two-factor.php

```diff
@@ -173,6 +173,13 @@ function wpcom_vip_enforce_two_factor_plugin() {
 			return $limited;
 		}, 9 );
 
+		// Calcuate two factor authentication support outside map_meta_cap to avoid callback loop
+		// see: https://github.com/Automattic/vip-go-mu-plugins/pull/1445#issuecomment-592124810
+		$is_user_using_two_factor = Two_Factor_Core::is_user_using_two_factor();
+		add_filter( 'wpcom_vip_is_user_using_two_factor', function() use ( $is_user_using_two_factor ) {
+			return $is_user_using_two_factor;
+		}, 9 );
```

Comment:
Minor thing: can we add a comment noting why we're using a priority of 9 here?

Reply:
@cojennin ^^^

Reply:
Ah my bad, I just copy and pasted the filter above, I'll remove the 9